docs/source/markdown/podman-play-kube.1.md

```diff
@@ -115,16 +115,20 @@ environment variable. `export REGISTRY_AUTH_FILE=path`
 
 #### **--build**
 
-Build images even if they are found in the local storage.
+Build images even if they are found in the local storage.  (This option is not available with the remote Podman client)
 
 #### **--cert-dir**=*path*
 
 Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry. (Default: /etc/containers/certs.d)
 Please refer to containers-certs.d(5) for details. (This option is not available with the remote Podman client)
 
+#### **--context-dir**=*path*
```

Comment:
Are these supposed to be listed alphabetically?

Reply:
This should be after _--configmap_, then.

Reply:
Moved